### PR TITLE
Fix mavutil imports

### PR DESCRIPTION
**Pull request by maxlou05**
mavutil is imported from pymavlink and not dronekit

### CODE REVIEW

File: image_encoding/test_image_encode_decode.py [hunk 1]
```diff
@@ -39,14 +39,3 @@ def main() -> int:
 
     # Note: the following fail since JPEG encoding is lossy
```
Comment (TongguangZhang) [inline image_encoding/test_image_encode_decode.py:40]: move this to docstring or conf maybe? up to you tho

File: image_encoding/test_image_encode_decode.py [hunk 2]
```diff
@@ -7,16 +7,16 @@
 from PIL import Image
 import numpy as np
 
-from image_encoding.modules import decoder
-from image_encoding.modules import encoder
+from .modules import decoder
+from .modules import encoder
```
Comment (TongguangZhang) [inline image_encoding/test_image_encode_decode.py:11]: similarly, import path from root

File: kml/test_ground_locations_to_kml.py [hunk 3]
```diff
@@ -6,8 +6,8 @@
 
 import pytest
 
-from kml.modules import ground_locations_to_kml
-from kml.modules import location_ground
+from .modules import ground_locations_to_kml
+from .modules import location_ground
```
Comment (TongguangZhang) [inline kml/test_ground_locations_to_kml.py:10]: same as above

File: mavlink/modules/flight_controller.py [hunk 4]
```diff
@@ -4,8 +4,9 @@
 
 import time
 
-from .. import dronekit
+from pymavlink import mavutil
```
Comment (TongguangZhang) [inline mavlink/modules/flight_controller.py:7]: why does our version of dronekit not include mavutil?

**Review by TongguangZhang — COMMENTED** [review]
reviewed

File: .github/workflows/run-tests.yml [hunk 5]
```diff
@@ -16,25 +16,30 @@ jobs:
...
 
+      # Upgrade pip
+      - name: Upgrade pip
+        run: |
+          python -m pip install --upgrade pip
+
       # Set up submodules
       - name: Set up all submodules and project dependencies
         run: |
-          git submodule update --init --recursive --remote
```
[the remark this answers is not in the document]
Comment (maxlou05) [reply .github/workflows/run-tests.yml:30]: Moved to line 21, during the checkout action

File: camera/test_camera.py [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 import cv2
 
-from camera.modules.camera_device import CameraDevice
```
[the remark this answers is not in the document]
Comment (maxlou05) [reply camera/test_camera.py:9]: See discussion comment in [Moved logger to common](https://github.com/UWARG/common/pull/45)

File: mavlink/modules/flight_controller.py (hunk 4 above)
Comment (maxlou05) [reply mavlink/modules/flight_controller.py:7]: I'm pretty sure dronekit never included mavutil. It's just that dronekit depended upon mavutil, and it somehow worked?

**Review by TongguangZhang — APPROVED** [review]
conditionally approved

**Pull request merged**